app/modules/transcribe/transcribe.controller.js

```diff
@@ -78,7 +78,8 @@ function TranscribeController($stateParams, $modal, $scope, $window, Annotations
     function loadSubject() {
         return SubjectsFactory.$getData($stateParams.subjectSet)
             .then(subjectLoaded, subjectLoadError)
-            .then(loadAggregations);
+            .then(loadAggregations)
+            .catch(handleErrors);
```

Comment:
This is catching any errors in `loadAggregations`, but not errors in loading subjects, because `subjectLoadError` already handled those.